src/renderer/components/dock/create-resource.tsx

```diff
@@ -30,11 +67,26 @@ export class CreateResource extends React.Component<Props> {
     return createResourceStore.getData(this.tabId);
   }
 
+  // templatesStores - provides list of folders with templates in priority order (first overrides others)
+  get templatesStores(): string[] {
+    const userStore = path.join((app || remote.app).getPath("userData"), "templates");
+    const stores = [];
+
+    if (fs.pathExistsSync(userStore)) stores.push(userStore);
```

Comment:
This could technically break in subtle ways. I would prefer just recreate it if not present since we handle the "no entries" case explicitly anyway.